extra/modules/greenbids-real-time-data/src/main/java/org/prebid/server/hooks/modules/greenbids/real/time/data/model/GreenbidsRealTimeDataProperties.java

```diff
@@ -2,26 +2,33 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.github.benmanes.caffeine.cache.Cache;
-import lombok.Value;
+import lombok.Data;
 import org.prebid.server.hooks.modules.greenbids.real.time.data.core.ThrottlingThresholds;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.v1.GreenbidsRealTimeDataModule;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
 
-import java.util.concurrent.locks.Lock;
 
-@Value(staticConstructor = "of")
+//@Value(staticConstructor = "of")
+//@NoArgsConstructor
+//@Builder(toBuilder = true)
+@ConfigurationProperties(prefix = "hooks.modules." + GreenbidsRealTimeDataModule.CODE)
+@Validated
```

Comment:
what's the purpose of the `@Validated` annotation?

Reply:
fixed, removed Validated